qemu_v8.mk

```diff
@@ -297,6 +325,37 @@ $(ROOTFS_UGZ): u-boot buildroot | $(BINARIES_PATH)
 				-n "Root file system" \
 				-d $(ROOTFS_GZ) $(ROOTFS_UGZ)
 
+################################################################################
+# XEN
+################################################################################
+.PHONY: xen
+xen-defconfig:
+	$(MAKE) -C $(XEN_PATH)/xen XEN_TARGET_ARCH=arm64 defconfig
+
+xen-common: xen-defconfig
+	cd $(XEN_PATH)/xen && \
+	tools/kconfig/merge_config.sh -m .config $(ROOT)/build/kconfigs/xen.conf
+
+xen: xen-common $(BINARIES_PATH)
+	$(MAKE) -C $(XEN_PATH) dist-xen \
+	XEN_TARGET_ARCH=arm64 \
+	CONFIG_XEN_INSTALL_SUFFIX=.gz	\
+	CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)"
+	ln -sf $(XEN_PATH)/xen $(BINARIES_PATH)
+
+XEN_TMP ?= $(BINARIES_PATH)/xen_files
+
+xen-create-image: $(XEN_TMP) xen linux buildroot
+	mkdir -p $(XEN_TMP)
+	cp $(KERNEL_IMAGE) $(XEN_TMP)
+	cp $(XEN_IMAGE) $(XEN_TMP)
+	cp $(XEN_CFG) $(XEN_TMP)
+	cp $(ROOT)/out-br/images/rootfs.cpio.gz $(XEN_TMP)
+	virt-make-fs -t vfat $(XEN_TMP) $(XEN_EXT4)
+
+xen-distclean:
+	$(MAKE) -C $(XEN_PATH) distclean
```

Comment:
Why `distclean`? Isn't `clean` good enough?

Reply:
clean should do the work. Will change.